examples/vl3_basic/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -25,7 +25,11 @@ spec:
     spec:
       containers:
         - name: vl3-nse
-          image: {{ .Values.registry }}/{{ .Values.org }}/vl3_ucnf-vl3-nse:{{ .Values.tag }}
+{{- if .Values.image }}
```

Comment:
IMO, having a variable image name is a bit too general and not needed in practice.  When you build the NSE docker image with `make docker-vl3` you don't need to modify the image name.  You can just use different values for TAG if you want to keep different versions of the image, e.g.:  `ORG=tiswanso TAG=vl3-lb-stats make docker-vl3`